src/TypeDefinition.js

```diff
@@ -123,6 +123,11 @@ export type HeaderConfig = {
    */
   style?: Object,
 
+  /**
+   * Style passed into navigation bar title
+   */
+  titleStyle?: Object,
```

Comment:
styles can be objects, numbers, booleans etc. so these ones look wrong. Maybe define a new type called style and reuse it.

```js
type Style = Object | number | false | void
```